flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/trait/AsyncOperatorModeTrait.java

```diff
@@ -0,0 +1,47 @@
+package org.apache.flink.table.planner.plan.trait;
```

Comment:
I think given where we left things with UNORDERED not covered in this first version, there's no need for sync mode and these trait classes and operator sync mode support can be removed for now.  What do you think @twalthr?

Reply:
Removed this